src/plugins/charts/public/static/components/_legend_toggle.scss

```diff
@@ -0,0 +1,30 @@
+.legend__toggle {
+  border-radius: $euiBorderRadius;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  padding: $euiSizeXS;
+  background-color: $euiColorEmptyShade;
+  transition: opacity $euiAnimSpeedFast $euiAnimSlightResistance, background-color $euiAnimSpeedFast $euiAnimSlightResistance $euiAnimSpeedExtraSlow;
+  z-index: 1;
+  margin: $euiSizeXS;
+
+  &:focus {
+    @include euiFocusRing();
+  }
+
+  &--isOpen {
+    background-color: transparentize($euiColorDarkestShade, .9);
+    opacity: 1;
+  }
+
+  &--position-left,
+  &--position-bottom {
+    left: auto;
```

Comment:
This looks suspicious to me... the class name is -left or -bottom, but the properties are right and top 0. 

Is this positioning prop still being used? If so, is something flipped here?